### PR TITLE
[terra-toggle]: Update terra-toggle animation to use 'ease-out' easing.

### LINKED ISSUE #2890: [terra-toggle] Update animation to ease-out
# Feature Request
Petition to update the animation on terra-toggle from `linear` to `ease-out`. 

> Easing out is typically the best for user interface work, because the fast start gives your animations a feeling of responsiveness, while still allowing for a natural slowdown at the end.

https://developers.google.com/web/fundamentals/design-and-ux/animations/the-basics-of-easing?hl=en

### PR DESCRIPTION
**Pull request by manjusr**
### Summary

Updated the animation on terra-toggle from ease to ease-out.

https://developers.google.com/web/fundamentals/design-and-ux/animations/the-basics-of-easing?hl=en#ease-out_animations 

Internal terra-toggle uses the react-animate-height component to perform toggle-animation when isAnimate is defined, by default, the 'ease' transition effect applied to the animation.
Added an 'ease-out' easing prop to the react-animate-height component to perform the ease-out transition effect on animation.

Closes #2890   

### Deployment Link

https://terra-core-update-terra-p2viph.herokuapp.com/home

### Testing

Ran and updated the jest tests.

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: packages/terra-toggle/src/Toggle.jsx [hunk 1]
```diff
@@ -31,10 +31,11 @@ const Toggle = ({
 }) => {
   const height = isOpen ? 'auto' : 0;
   let body;
+  const easing = "ease-out";
 
   if (isAnimated) {
     body = (
-      <AnimateHeight duration={250} height={height}>
+      <AnimateHeight duration={250} height={height} easing={easing}>
```
Comment (emilyrohrbough) [inline packages/terra-toggle/src/Toggle.jsx:38]:
```suggestion
      <AnimateHeight duration={250} height={height} easing="ease-out">
```

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
@manjusr Thank you for the contribution!! 🎉

**Comment by yuderekyu** [comment]
@manjusr i'm not sure why the travis build isn't starting. can you push emily's suggested change to see if we can re-trigger the build?

File: packages/terra-toggle/src/Toggle.jsx [hunk 2]
```diff
@@ -31,10 +31,11 @@ const Toggle = ({
 }) => {
   const height = isOpen ? 'auto' : 0;
   let body;
+  const easing = "ease-out";
```
Comment (yuderekyu) [inline packages/terra-toggle/src/Toggle.jsx:34]:
```suggestion
```

**Comment by manjusr** [comment]
> Thank you for the contribution!!

It is my pleasure to contribute here.

**Comment by manjusr** [comment]
> @manjusr i'm not sure why the travis build isn't starting. can you push emily's suggested change to see if we can re-trigger the build?

@yuderekyu I could see the Travis build after committing the changes. Thanks for your timely help!!

**Review by mjhenkes — APPROVED** [review]

**Review by jeremyfuksa — APPROVED** [review]

**Comment by mjhenkes** [comment]
Thanks for the contribution @manjusr !

**Pull request merged**